src/hidra/sender/eventdetectors/watchdog_events.py

```diff
@@ -489,7 +502,23 @@ def check_last_modified(self, filepath):
             return
 
         # compare ( >= limit)
-        if time_current - time_last_modified >= self.time_till_closed:
+        if (time_current - time_last_modified >= self.time_till_closed * 4 and file_size == 0 ):
+            self.log.debug("0kb file detected: %s", filepath)
+            
+            event_message = get_event_message(filepath, self.mon_dir)
+            self.log.debug("event_message: %s", event_message)
+
+            # add to result list
+            _event_store.add(event_message)
+
+            with self.lock:
+                _events_marked_to_remove.append(filepath)
+        else:
+            self.log.debug("File was last modified %s sec ago: %s",
+                           time_current - time_last_modified, filepath)
```

Comment:
Reuse the existing code instead of copying it.

Reply:
I have created a function now to tackle this.